### DISCUSSION TITLE
wind sound in headphones, part 2

### DISCUSSION THREAD

**Discussion by mariakei** [jamulussoftware/jamulus#2718] [Technical Support & Help] [answered]
Hi, 

Every timeI use jamulus, I get a persistent wind sound in my headphones. It's there regardless of the strength of the connection. And all the other musicians in our jamulus room hear it too. 

I saw there was someone with the same issue who posted in 2017, but it didn't get resolved (publicly, at least). Attached is a screenshot of that discussion. Also can be found at this link: https://sourceforge.net/p/llcon/discussion/hardware/thread/509279c9/ 

Anyone have any ideas?? I'm getting desperate. I AM using the Asio4All driver.

![image](https://user-images.githubusercontent.com/108960422/178081041-79a4dbaa-459e-4021-acb4-721d24a71d0d.png)

**Comment by ann0see** [comment]
Probably the issue is ASIO4ALL. At least it's known to sometimes cause strange issues. The only thing I could suggest would be to play around with the settings, contact the developer or better buy a cheap audio interface.

**Comment by dcorson-ticino-com** [comment]
Go for an audio interface, but be sure it has its own ASIO driver and doesn't use ASIO4ALL.
ASIO4ALL is the main problem for Windows users.
Don

**Comment by pljones** [comment]
Can you say more about your set up, so we can help you.

Things like
- do you have a microphone?
- do you have an external microphone and a built in microphone?
- what are your ASIO4ALL settings? (Advanced mode screen shot)
- what are your Jamulus audio settings?
- what can you hear when you are not connected to a server?

There's so much you've not said, no one can really help as it stands.

**Comment by mariakei** [comment]
Thanks all for weighing in.
 Yes, I have a microphone. i'm using a wired headphone set that has a
microphone on the wire itself. There is a built in microphone on the
computer, but if I understand correctly, the microphone on my headphones
are the ones I'm using.

Here are some screenshots of my ASIO4ALL settings:

[image: image.png]

[image: image.png]

And jamulus audio settings:
[image: image.png]
I've tried playing around with different audio settings features (changing
the buffer delay, etc.) but it makes no difference.

When I'm not connected to a server, I hear silence. (But I know the
headphones/computer sound system is not the issue, because I can hear audio
on a youtube video, for example, just fine.)

Thanks for your help!


<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Sat, Jul 9, 2022 at 12:44 AM Peter L Jones ***@***.***>
wrote:

> Can you say more about your set up, so we can help you.
>
> Things like
>
>    - do you have a microphone?
>    - do you have an external microphone and a built in microphone?
>    - what are your ASIO4ALL settings? (Advanced mode screen shot)
>    - what are your Jamulus audio settings?
>    - what can you hear when you are not connected to a server?
>
> There's so much you've not said, no one can really help as it stands.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/jamulussoftware/discussions/2718#discussioncomment-3111712>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ7JVJVCIM2UF6H4CU44AT3VTEUVFANCNFSM53CMLDZA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Maria Kei Oldiges
845-489-4780 (c)
***@***.***

**Reply by dcorson-ticino-com** [reply]
You say the headphones is good, but what about the mic ?
Try using the internal mic to see if there is a difference.

**Reply by pljones** [reply]
Hi @mariakei ,

Sorry but your screenshots aren't showning for me: I just see three copies of "[image: image.png]".

**Reply by pljones** [reply]
I'll also add:
> (But I know the headphones/computer sound system is not the issue, because I can hear audio on a youtube video, for example, just fine.)

This doesn't really help with Jamulus as it doesn't necessarily use the same audio system as you use for watch YouTube videos.

**Comment by ghost** [comment]
> When connecting to a Jamulus server, I can hear Mic and Bass, however there is a pulsating/wind noise while playing or talking.
>
Could be subsonic noise; filter it if you can, or use different equipment.
Could be an old version of Jamulus.

**Reply by thetome** [reply]
You could take a look at Windows sound setup, and see if one or more recording devices from your PC have the tickbox checked 'Use this device as playback source' (translated from my German Windows, may read a bit different on your PC), and further check if these devices are activated in your Jamulus ASIO4ALL settings too.

![grafik](https://user-images.githubusercontent.com/98780189/183377330-5003f549-9f98-4a2c-b623-eb309d0ceb8d.png)

**Comment by mariakei** [comment, marked as answer]
Hi, just wanted to update this thread with the resolution in case others in
the future encounter this issue (of a "wind/ocean" sound in the audio). I
bit the bullet and bought an audio interface (the scarlett solo) and it
100% resolved the issue. Not sure what that says about the
problem--probably that the built in audio on my laptop was subpar. Thanks
all for your help.


On Mon, Aug 8, 2022 at 1:44 AM thetome ***@***.***> wrote:

> You could take a look at Windows sound setup, and see if one or more
> recording devices from your PC have the tickbox checked 'Use this device as
> playback source' (translated from my German Windows, may read a bit
> different on your PC), and further check if these devices are activated in
> your Jamulus ASIO4ALL settings too.
>
> [image: grafik]
> <https://user-images.githubusercontent.com/98780189/183377330-5003f549-9f98-4a2c-b623-eb309d0ceb8d.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/jamulussoftware/discussions/2718#discussioncomment-3347578>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ7JVJQSZWPFNQGWMRS6QFDVYDCGDANCNFSM53CMLDZA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
Maria Kei Oldiges
845-489-4780 (c)
***@***.***

**Reply by ann0see** [reply]
Good to know. Inbuilt mics and soundcards are often not studio quality unfortunately, so even cheap interfaces give a big benefit.

**Reply by thetome** [reply]
> Hi, just wanted to update this thread with the resolution in case others in the future encounter this issue 

Nice move, thank you for the feedback. Same experience here, in the beginning I fiddled around with onboard/cheap soundcards with generally frustrating results. Any interface with an own ASIO driver will do better.
